Add log pipeline (#8068)
DataDog/integrations-core: aerospike/README.md

#### Host

##### Metric collection
To configure this check for an Agent running on a host:

1. Edit the `aerospike.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your aerospike performance data. See the [sample aerospike.d/conf.yaml][1] for all available configuration options.

2. [Restart the Agent][2].

##### Log collection

> _Available for Agent versions >6.0_

Reviewer: ```suggestion
```

1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

   ```yaml
   logs_enabled: true
   ```

> 2. Add this configuration block to your `aerospike.d/conf.yaml` file to start collecting your Apache Logs:

Reviewer: ```suggestion
2. Add this configuration block to your `aerospike.d/conf.yaml` file to start collecting your Aerospike Logs:
```

   ```yaml
   logs:
     - type: file
       path: /var/log/aerospike/aerospike.log
       source: aerospike

>        service: <SERVICE_NAME>

Reviewer: ```suggestion
```

   ```

>     Change the `path` and `service` parameter values and configure them for your environment. See the [sample aerospike.d/conf.yaml][1] for all available configuration options.

Reviewer: ```suggestion
    Change the `path` parameter value and configure them for your environment. See the [sample aerospike.d/conf.yaml][1] for all available configuration options.
```
